ldap/servers/slapd/slapi-plugin.h

```diff
@@ -6977,50 +6974,6 @@ slapi_timer_result slapi_timespec_expire_check(struct timespec *expire);
                                          * into tombstone
                                          */
 
-/*
```

Comment:
I do not like the massive change in slapi-plugin.h 
  (looks like a risk of breaking existing custom plugin
    And after a closer look as you changed the #define values, it will definitively break the binary compatibility
     with existing custom plugin.
IMHO we should not do this. we should be able to split the switches without changing slapi-plugin.h

Reply:
an option would be to have a conversion table that convert a former define value into an index into the callback table

Reply:
FYI, or you can do a simple table:
 here is a python script that generate a skeleton of what it would looks like:
[split_pblock_switches.py.txt](https://github.com/user-attachments/files/18657166/split_pblock_switches.py.txt)